html/admin/api.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
I think the php-fpm process needs to be restarted. www-data is in the Bible box group, which is configured in sudoers.